migrations/20220615150000-add-application-id-to-activities.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn('Activities', 'ApplicationId', {
```

Comment:
@Betree I thought about that, but I'm still not sure why an application would have multiple UserTokenIds for the same user, meaning that it would still be possible to implement https://github.com/opencollective/opencollective/issues/5650 by tracking only `ApplicationId`.

I do see an advantage in tracking which session the user is using tho. I'll refactor this PR.